product-scenarios/2-Bridging-systems-that-communicate-in-different-protocols/02-SynapseConfigProject/02-synapseConfig/src/main/synapse-config/endpoints/HttpMessageWithoutAlteringDownstreamTestAddressEP.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shall we add the license header? Shall we update other files as well?

Reply:
done